### DISCUSSION TITLE
Frontend Initiatives Meeting: February 3, 2021

### DISCUSSION THREAD

**Discussion by varl** [dhis2/notes#202] [Meetings] [answered]
When: February 3, 2021 @ 15:00 CET
Where: Remote

Agenda

- General updates
  - Team Platform is introducing a rotation for first-line support: https://github.com/dhis2/notes/pull/201
    - Instead of asking questions on Slack, please use the Q&A category here on [dhis2/notes/discussions](https://github.com/dhis2/notes/discussions/categories/q-a)
- Working group updates
  - Cypress (@HendrikThePendric @Mohammer5)
  - PWA & Offline/Responsive Dashboard (@jenniferarnesen @Sharmyn28 @amcgee)
  - Plugin builds (@amcgee @jenniferarnesen @edoardo)
  - UI & Platform updates (@ismay @HendrikThePendric @cooper-joe @amcgee @edoardo)
  - Developer outreach & resources - portal and community plans #213 @deboragaleano 
- [Dynamic runtime modules](https://www.dropbox.com/s/eum0j1s4mdnc0es/Dynamic%20Modules%20and%20Plugins%20-%20Spec.pdf?dl=0) - [SystemJS demo](https://romantic-morse-3fe5bf.netlify.app/) (@amcgee)
- Others?

**Comment by amcgee** [comment]
@varl I added dev outreach (@deboragaleano can share her proposal)

**Reply by deboragaleano** [reply]
Thank you! :)

**Comment by ghost** [comment]
Proposal: a dedicated google calendar for the helpdesk rotation, to show people who's on duty, and help remind the people who are on duty.

**Comment by varl** [comment, marked as answer]
Here are the approved notes: https://github.com/dhis2/notes/blob/master/decisions/2021/02/03-fe-meeting.md
